### PR TITLE
feat: Edit/delete Program, Projects and Tasks

### LINKED ISSUE #31: Program Page
- [x] Table shows all Projects and nested Tasks
- [x] Table is inline editable. Data is saved on unfocus for text fields, or on select for select/date fields.
- [ ] Clicking Task Title navigates to Task Details page
- [x] Add Project
- [x] Add Task
- [x] Delete Task
- [ ] Tests
- [x] On page load, data is retrieved from DynamoDB for that Program including all Projects and Tasks
- [ ]  On login, if the user has a single Program, they should be dropped on that program page.
- [x] Add AppSync endpoint
- [x] Add DynamoDB Tables for Programs and Tasks

Not required for MVP:

- [ ] Edit Project
- [ ] Delete Project
- [ ] Change owner - don't show the Owner column for MVP since this is always the same

### PR DESCRIPTION
**Pull request by proserve**
*close #31*

*Description of changes:*
**Frontend**: 
- Edit/Delete Program
- Edit/delete Project
- Edit/delete task

**Backend**
- Delete Program API
- Delete Task API
- Delete Project API

Screen Shot 2021-04-24 at 12 23 24 AM

# Checklist

- [ ] Tests have been added and are passing
- [ ] Documentation has been updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/api/models/Disease.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const Disease = new Entity({
     omimId: 'string',
     causalGene: 'string',
     mutationImpact: 'string',
-    proteinSize: 'number',
+    proteinSize: 'string',
```
Comment (brettstack) [inline packages/api/models/Disease.js:21]: Why the change?

**Review by brettstack — APPROVED** [review]

File: packages/api/models/Disease.js (hunk 1 above)
Comment (proserve) [reply packages/api/models/Disease.js:21]: These are the possible values: 
Screen Shot 2021-04-27 at 12 01 35 AM
